install/install-apt-deps.sh

```diff
@@ -31,4 +30,5 @@ sudo apt install -y nodejs;
 sudo apt install -y npm;
 sudo npm install -g pm2;
 
-sudo apt install -y awscli;
\ No newline at end of file
+sudo apt install -y awscli;
+sudo apt install -y pkg-config;
```

Comment:
pkg-config is required for `make build`

Reply:
could we just put review comments like these in directly as code comments?